seatunnel-connectors-v2/connector-doris/src/main/java/org/apache/seatunnel/connectors/doris/sink/writer/DorisSinkWriter.java

```diff
@@ -105,9 +98,8 @@ public DorisSinkWriter(
         this.initializeLoad();
     }
 
-    private void initializeLoad() throws IOException {
-        this.backends = RestService.getBackendsV2(dorisConfig, log);
-        String backend = getAvailableBackend();
+    private void initializeLoad() {
+        String backend = dorisConfig.getFrontends();
```

Comment:
It looks like it just returns fe. I wonder what's the purpose here?

Reply:
This is this PR purpose. Change stream load endpoint from BE to FE.

Reply:
Fe is mainly responsible for doris's query plan. I am worried that if stream load also selects fe, the efficiency of fe will be affected in high concurrency scenarios.

Reply:
Never mind. FE will use http redirect, then data also send from SeaTunnel to BE.
![image](https://github.com/apache/seatunnel/assets/32387433/01f784cc-e29a-4e0f-ba1c-fa03f52edef2)